BoxContentSDK/BoxContentSDK/Requests/BOXFolderPaginatedItemsRequest.h

```diff
@@ -15,6 +15,11 @@
 //can be used as a way to link related paginated requests together
 @property (nonatomic, readwrite, strong) NSData *sharedPaginatedRequestData;
 
+//Metadata information parameters
+@property (nonatomic, readwrite, strong) NSString *templateKey;
+@property (nonatomic, readwrite, strong) NSString *scope;
```

Comment:
What is scope?

Reply:
These should be `copy`, rather than `strong`. This applies basically to all `NSString`, `NSArray`, `NSData`, etc.